bin/bundle

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env ruby
-ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile', __FILE__)
```

Comment:
Most of the following set of changes were made automatically by rails on the upgrade to 5.2